subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -106,9 +105,7 @@ class AndroidSantaTrackerSmokeTest extends AbstractAndroidSantaTrackerSmokeTest
     }
 
     @Unroll
-    @UnsupportedWithConfigurationCache(iterationMatchers = [AGP_3_ITERATION_MATCHER, AGP_4_0_ITERATION_MATCHER, AGP_4_1_ITERATION_MATCHER, AGP_4_2_ITERATION_MATCHER])
```

Comment:
:question: I would keep that annotation. Lint isn't supported with the configuration cache and should get fixed in AGP 7.

Reply:
I re-added the annotation. I removed it since I thought the lint task doesn't work with all the older AGP versions at all, though it actually should work with AGP 4.2, so the `UnsupportedWithConfigurationCache` still makes sense.

Reply:
If this test only runs on >=4.2 then the list of `iterationMatchers` here could be reduced

Reply:
I removed the unused iteration matchers.